src/platform/platform.c

```diff
@@ -0,0 +1,10 @@
+#include "platform.h"
+
+#include "SDL.h"
+
+int platform_version_at_least(int major, int minor, int patch)
```

Comment:
I know you want to hide SDL from the files, but I still think the name `platform_sdl_version_at_least` would be clearer.

Reply:
Fair point, will rename.